routing/route.go

```diff
@@ -34,6 +35,35 @@ type Hop struct {
 	AmtToForward lnwire.MilliSatoshi
 }
 
+func serializeHop(w io.Writer, h *Hop) error {
```

Comment:
Perhaps this should be pushed down into the `channeldb` level? Or perhaps it's meant to be used for unit tests down the line.

Reply:
It is only used to serialize the route in the `payment state machine` for now, so I think it makes sense to keep it here, where `Hop` is defined.